spot-vaults/contracts/BillBroker.sol

```diff
@@ -77,6 +77,7 @@ contract BillBroker is
     uint256 public constant DECIMALS = 18;
     uint256 public constant ONE = (10 ** DECIMALS);
     uint256 private constant INITIAL_RATE = 1000000;
+    uint256 public constant MINIMUM_LIQUIDITY = 10 ** 22;
```

Comment:
Based on our implementation on first deposit, 1e24 LP tokens  are basically 1$ worth.

Thus 1e22 = 0.01$  which is the amount we lock into the contract forever.